tests/src/common/test_case/mod.rs

```diff
@@ -49,6 +50,10 @@ type OptionalOracleWrapper = Option<
     >,
 >;
 
+type Lpn = Usdc;
+pub const MIN_ASSET: Coin<Lpn> = Coin::<Lpn>::new(MIN_ASSET_AMOUNT);
+pub const MIN_SELL_ASSET: Coin<Lpn> = Coin::<Lpn>::new(MIN_SELL_ASSET_AMOUNT);
```

Comment:
1. `::<Lpn>`  is redundant
2. it's a copy/paste of the same from above, do we need another pair?